plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -248,7 +243,7 @@ def _get_cli_vars(self) -> str:
             (self.templater_selector, self.name, "context")
         )
 
-        return str(cli_vars) if cli_vars else "{}"
+        return cli_vars if cli_vars else {}
```

Comment:
I'm confused about this change. The function type says it returns `str`, but we removed a `str()` conversion as well as changing the `else` to return a dictionary. Why make this change? Is it related to the dbt issue?

Reply:
Yeah, it wants this through as a dict now. I've just committed a change to the type hint